### PR TITLE
Exclude orderBy createdDateTime in case of hasDisplayAggregate : true and no other orderBy applied

### PR DESCRIPTION
**Pull request by anant-dev**
PR  regarding - Collection bug fix - when you have a display aggergate and no order by then exclude order by createDatetime from being added , Task on Teamwork , @rmarchandTen24  please review

### CODE REVIEW

File: model/entity/Collection.cfc [hunk 1]
```diff
@@ -2835,7 +2843,9 @@ component displayname="Collection" entityname="SlatwallCollection" table="SwColl
 				}
 
 				if(!structKeyExists(collectionConfig,'orderBy') || !arrayLen(collectionConfig.orderBy)){
-					arrayAppend(groupBys,'_' & lcase(getService('hibachiService').getProperlyCasedShortEntityName(getCollectionObject())) & '.' & "createdDateTime");
+					if(!getHasDisplayAggregate()){
+						arrayAppend(groupBys,'_' & lcase(getService('hibachiService').getProperlyCasedShortEntityName(getCollectionObject())) & '.' & "createdDateTime");
```
Comment (rmarchandTen24) [inline model/entity/Collection.cfc]: @anant-dev can you change this line to use the default order by vs createdDateTime?

Comment (rmarchandTen24) [reply model/entity/Collection.cfc]: getDefaultOrderBy()

Comment (rmarchandTen24) [reply model/entity/Collection.cfc]: @migueltarga I think if the default orderby is not createdDatetime then we get a mismatch on the group by

Comment (migueltarga) [reply model/entity/Collection.cfc]: @rmarchandTen24 you are right about that! Good catch

Comment (anant-dev) [reply model/entity/Collection.cfc]: @rmarchandTen24  yes you are right, am replacing it by getDefaultOrderBy().propertyIdentifier

**Comment by anant-dev** [comment]
@rmarchandTen24 made changes as per the suggestion in the review. Thanks for pointing it out.

**Comment by anant-dev** [comment]
@rmarchandTen24  please review the PR and the **latest commit adding tests to check if orderBy createdDateTime is excluded in case of hasDisplayAggregate is true when orderBy applied or not applied**

**Pull request merged**